omnibus/files/private-chef-cookbooks/private-chef/spec/libraries/helper_spec.rb

```diff
@@ -114,6 +114,64 @@
     end
 
     context 'when elastic search is unavailable' do
+      it 'should return 2 but the request fails first time only' do
```

Comment:
Shared context has always seemed like wonky rspec voodoo to me, so I think it's just a matter of taste. :)